OutOfSchool/OutOfSchool.Encryption/Config/Proxy.cs

```diff
@@ -0,0 +1,14 @@
+namespace OutOfSchool.Encryption.Config;
+
+public class Proxy
+{
+    public bool Use { get; set; }
+
+    public string Address { get; set; }
+
+    public string Port { get; set; }
```

Comment:
Is it correct ot have `Port` as a string?

Reply:
And another question:
I haven't seen the validation anywhere... 
Could you please add the check whether the port is valid ? (>=0 && < ushort.MaxValue ) ?

Reply:
Port as string is because the `C` library methods have ports as string. I can add validation, but is it necessary to dot `string -> int -> string` convertion?

Reply:
Types are part of self-documented code. In other words, I don't expect `":8080"` value in `int` variable. Thus I'm ready to pay this "double conversion" price. 
Annotation you've used here is good as well for the mentioned purpose (though I more trust to compiler than some reflection).